### PR TITLE
Add new monitoring label

### PR DESCRIPTION
**Pull request by kopiczko**
Towards https://github.com/giantswarm/giantswarm/issues/14586

### CODE REVIEW

**Review by corest — APPROVED** [review]

File: helm/config-controller/templates/service.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: {{ include "resource.default.namespace"  . }}
   labels:
     {{- include "labels.common" . | nindent 4 }}
+    giantswarm.io/monitoring: "true"
```
Comment (QuentinBisson) [inline helm/config-controller/templates/service.yaml:8]: You also need to configure `giantswarm.io/monitoring-path" and "giantswarm.io/monitoring-port" for the new setup to pick this up

Comment (kopiczko) [reply helm/config-controller/templates/service.yaml:8]: I thought there are defaults?

Comment (QuentinBisson) [reply helm/config-controller/templates/service.yaml:8]: Sadly no, we have a stuck PR about this: https://github.com/giantswarm/prometheus-meta-operator/pull/245

Comment (kopiczko) [reply helm/config-controller/templates/service.yaml:8]: TBH I even prefer to be explicit. I'll fix it.

**Pull request merged**